localstack/services/transcribe/provider.py

```diff
@@ -333,3 +333,5 @@ def _run_transcription_job(self, args: Tuple[TranscribeStore, str]):
             job["TranscriptionJobStatus"] = TranscriptionJobStatus.FAILED
 
             LOG.warning("Transcription job %s failed: %s", job_name, job["FailureReason"])
+
+            raise exc
```

Comment:
Imho, a comment or a slight change would be nice here.
The exception here is only raised such that it's stacktrace is shown in the logs (because this function is executed asynchronously in another thread)? Or is it caught anywhere?
If it's just about the logging of the stacktrace, and you need it everytime you end up here, you could just use `LOG.exception` in the line above: https://docs.python.org/3/library/logging.html#logging.Logger.exception

Reply:
> The exception here is only raised such that it's stacktrace is shown in the logs (because this function is executed asynchronously in another thread)? Or is it caught anywhere?

Yes that's the intention here -- having the stack trace appear to help troubleshoot why the transcription job failed. The exc is not caught anywhere.

Reply:
Ah, okay. Then I would suggest just switching `LOG.warning(...)` with `LOG.exception(...)` in line 335.
This would clearly state what's happening.

Reply:
I see what you meant. Fixed in f267ded9 :+1: